Patches/SQLite3/SQLite3Config.cmake.in

```diff
@@ -0,0 +1,10 @@
+# Config file used to import the SQLite3 build tree.
+set( SQLite3_INCLUDE_DIR "@CMAKE_INSTALL_PREFIX@/include" )
+set( SQLite3_LIBRARY sqlite3 )
```

Comment:
Shouldn't this be a path? Maybe we need to extract the target file name in the `CMakeLists.txt`? (Or use `find_library` here?)